cluster/examples/kubernetes/ceph/images.txt

```diff
@@ -2,8 +2,8 @@
  quay.io/ceph/ceph:v16.2.5
  quay.io/cephcsi/cephcsi:v3.4.0
  k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.0
- k8s.gcr.io/sig-storage/csi-resizer:v1.2.0
  k8s.gcr.io/sig-storage/csi-provisioner:v2.2.2
- k8s.gcr.io/sig-storage/csi-snapshotter:v4.1.1
  k8s.gcr.io/sig-storage/csi-attacher:v3.2.1
+ k8s.gcr.io/sig-storage/csi-snapshotter:v4.1.1
```

Comment:
The order just changed because it switched from the yaml to the code, right? Just checking that the txt file will be consistently ordered.

Reply:
right, it changed because we switched from yaml to code